sftp-connector/processes/Sftp/SftpDownloadFile.p.json

```diff
@@ -59,7 +59,8 @@
             "  ivy.log.debug(\"The following file: {0} will be downloaded from the server. Size: {1} bytes\", in.remoteFileName, fileData.size);",
             "  ",
             "  in.toFile = new File(in.remoteFileName, true);",
-            "",
+            "  in.toFile.getParentFile().mkdir();",

```

Comment:
should we check if the parent dir exists  before creating it?

Reply:
@ntqdinh-axonivy Yes, because when we download many files from a folder, the creation command is called once.
I have updated this. 
Could you have a look ?